docs/notebooks/markov-models.ipynb

```diff
@@ -137,7 +137,7 @@
     "The symbol etymology, which usually gets swept under the rug in mathematically-oriented papers, are as follows: \n",
```

Comment:
<ul><li>I'm wondering if the states plots would be clearer as scatters instead of lines?</li><li>As the number of states is an integer, I think these plots would be clearer if you restricted the y-axis tick labels to these values (0, 1, 2).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ericmjl/bayesian-analysis-recipes/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>I think it's clearer as lines, as they clearly delineate the three states. Agreed, though, that the yticks could be clearer.</p>